### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by UserConor**
Added the ability to view a delivery employee via their ID.

### CODE REVIEW

**Review by BlankAmber — APPROVED** [review]
MOST SUPREME & SUBLIME

File: src/main/java/org/kainos/ea/resources/DeliveryEmployeeController.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import javax.ws.rs.core.Response;
+
+@Api("Dropwizard Delivery Employee API")
+@Path("/api")
+public class DeliveryEmployeeController {
+
+        private DeliveryEmployeeService deliveryEmployeeService = new DeliveryEmployeeService();
+
+        @GET
+        @Path("/deliveryemployees")
```
Comment (jkelso662) [inline src/main/java/org/kainos/ea/resources/DeliveryEmployeeController.java:22]: Looks like the naming conventions of the paths have strayed a bit. I think I saw kebab case being used elsewhere

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Good pull request. Just one naming issue - then all good!

**Review by jkelso662 — APPROVED** [review]
Nice!

**Pull request merged**